openshift/olm/knative-serving.catalogsource.yaml

```diff
@@ -280,19 +280,21 @@ data:
                         value: image-registry.openshift-image-registry.svc:5000/knative-serving/knative-serving-activator
                       - name: IMAGE_autoscaler
                         value: image-registry.openshift-image-registry.svc:5000/knative-serving/knative-serving-autoscaler
+                      - name: IMAGE_autoscaler-hpa
+                        value: image-registry.openshift-image-registry.svc:5000/knative-serving/knative-serving-autoscaler-hpa
                       - name: IMAGE_controller
                         value: image-registry.openshift-image-registry.svc:5000/knative-serving/knative-serving-controller
                       - name: IMAGE_networking-certmanager
                         value: image-registry.openshift-image-registry.svc:5000/knative-serving/knative-serving-certmanager
                       - name: IMAGE_networking-istio
                         value: image-registry.openshift-image-registry.svc:5000/knative-serving/knative-serving-istio
-                      - name: IMAGE_networking-nscert
+                      - name: IMAGE_networking-ns-cert
                         value: image-registry.openshift-image-registry.svc:5000/knative-serving/knative-serving-nscert
                       - name: IMAGE_webhook
                         value: image-registry.openshift-image-registry.svc:5000/knative-serving/knative-serving-webhook
                       image: quay.io/openshift-knative/knative-serving-operator:v0.8.1-1.1.0-04
                       args:
-                        - --filename=https://raw.githubusercontent.com/openshift/knative-serving/release-v0.9.0/openshift/release/knative-serving-ci.yaml # remove this from individual release branches
+                        - --filename=https://raw.githubusercontent.com/openshift/knative-serving/release-v0.9.0/openshift/release/knative-serving-v0.9.0.yaml # remove this from individual release branches
```

Comment:
I see file name is still [openshift/release/knative-serving-ci.yaml](https://github.com/openshift/knative-serving/pull/276/files#diff-9d9ad9c99bcf6aa573966c3ee4ab0d0c)

Reply:
That is fine, the commit touching that file merely fixes it to be complete. We could theoretically delete it, would yo prefer that?

Reply:
Yep that should  be fine.
When are we planning to remove that?

Reply:
Well we can remove them from the individual release branches immediately. `release-next` will still it.

Reply:
In fact, the release branches need it as well because the script reads it to get to know which images to make available in the cluster. We should break that apart first but I'd rather not do this in this PR so I'd actually keep it as-is for now.